src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin field($size) {
+  width: $size;
+  height: $size;
+  box-sizing: border-box;
+  border: $field-border solid black;
+  background-color: #eee;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition-duration: 0.5s;
```

Comment:
- Use transition property here
- List all properties that you apply transition to. It will be easier to understand which properties are affected by it and also there will be no case in which something that should not have a transition has one